### DISCUSSION TITLE
Tried to cheat... didn't work as expected 😅😅😅

### DISCUSSION THREAD

**Discussion by joewinbpaul** [cssbattle/forum#334] [General]
I got a weird idea of _cheating in cssbattle_ and I tried it.  <p> I used `background-image` to get 100% with low char count  <p> For Eg: <p> `<style>*+*{background:url(https://cssbattle.dev/targets/29.png` <p> I was able to submit it.. but then this happened: 
![Screenshot (1413)](https://user-images.githubusercontent.com/102139966/208301210-e2fc3a96-7c0d-44ab-8007-dbd0e9e35347.png)
<p> `You scored 0 with 0% match... `<p> Definitely unexpected...

**Comment by joewinbpaul** [comment]
Don't ask about the other tabs I had opened up.....

**Reply by joewinbpaul** [reply]
> Don't ask about the other tabs I had opened up.....

And bookmarks 😅

**Comment by Beowolve** [comment]
That target did drive me crazy sometimes, I can see why u tried that :D

![image](https://user-images.githubusercontent.com/32326640/208642932-edfd9d7a-5d4b-4582-af38-51a88eda986b.png)

**Reply by joewinbpaul** [reply]
> That target did drive me crazy sometimes, I can see why u tried that :D
> 
> ![image](https://user-images.githubusercontent.com/32326640/208642932-edfd9d7a-5d4b-4582-af38-51a88eda986b.png)

I hate that specific target because it wants the code to be pixel correct. And even tho it did become px correct.. it says that I got a 99.9% match <p> So far I didn't get a single 100% solution on that target...

**Reply by SelenIT** [reply]
This box-shadow version is 100%, I've just checked:
```
<body
 bgcolor=1a434
 text=f3ac3c
 style=margin:-75%50%75%-25%;border-radius:25vw;box-shadow:0+50vh,0+150vh,75vw+50vh,75vw+150vh>
```

Radial gradient approach works too, but yes, it requires fine tuning the smoothing of the color edge with the precision up to 0.1px, our common way of making abrupt color changes like `#1a4341 106q, #f3ac3c 0` doesn't work here :(

**Reply by joewinbpaul** [reply]
thx!

**Reply by joewinbpaul** [reply]
> thx!

I finally got a 100% match!
